README.md

```diff
@@ -1,2 +1,48 @@
 # Tailwind CSS Plugins
 
+This plugin wraps up a collection of other plugins we've written for [Tailwind CSS](https://tailwindcss.com/). Plugins include:
+
+* blend
+* flex-basis
+* parent-expanded
+* parent-open
+* rect
+
+## Installation
+
+```
+# install via npm
+npm i @viget/tailwindcss-plugins -D
+
+# install via yarn
+yarn add @viget/tailwindcss-plugins -D
+```
+
+## Usage
+
+Simply register the available plugins in your tailwindcss.config.js file, and follow the usage instructions for each [plugin](plugins) above.
+
+```
+plugins: [
+  require('@viget/tailwindcss-plugins/PLUGIN_NAME'),
+  // ...
+],
+```
+
+## Notes
+
+Additionally, there are some other plugin examples included in this repository in the **other** directory. These are not registerable in your Tailwind config. The reasons for this are that these represent plugins which:
```

Comment:
do you think `examples` would better describe this dir?

Reply:
Yeah I think so.